modules/metallb/templates/config.yaml.tftpl

```diff
@@ -1,3 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
```

Comment:
For a follow-up PR (since this is hard-coded throughout the MetalLB template):  are we good to keep the MetalLB namespace / and resource names hard-coded, or do we need to provide more flexibility for there?

Reply:
I think we should, yes, but it's definitely for a follow up PR.